package/MDAnalysis/analysis/align.py

```diff
@@ -652,15 +653,18 @@ def rms_fit_trj(
          - ``True``: suppress progress and logging for levels INFO and below.
          - ``False``: show all status messages and do not change the the logging
            level (default)
-
+      *in_memory*
```

Comment:
I haven't tested the speed very systematically for the alignment, so I've just added ", which can improve performance substantially in some cases".